sys/include/net/nanocoap.h

```diff
@@ -395,6 +395,22 @@ static inline unsigned coap_get_token_len(const coap_pkt_t *pkt)
     return (pkt->hdr->ver_t_tkl & 0xf);
 }
 
+/**
+ * @brief   Get pointer to a message's token
+ *
+ * @param[in]   pkt   CoAP packet
+ *
+ * @returns     pointer to the token, or NULL if there is no token
```

Comment:
```suggestion
 * @returns     pointer to the token-position
```
does not tell you about its lenght